packages/cli/src/main_parser_factory.ts

```diff
@@ -29,6 +30,7 @@ export const createMainParser = (libraryVersion: string): Argv => {
     .command(createConfigureCommand())
     .command(createInfoCommand())
     .help()
+    .alias('h', 'help')
```

Comment:
could you add test coverage for the alias here https://github.com/aws-amplify/amplify-backend/blob/main/packages/cli/src/main_parser_factory.test.ts ?

Reply:
added the tests for the shorthand versions